src/app/common/theme-provider.tsx

```diff
@@ -27,12 +30,24 @@ const ThemeContext = createContext<{
 const getSystemTheme = () =>
   window.matchMedia('prefers-color-scheme: dark').matches ? 'dark' : 'light';
 
+function getComputedTheme(userSelectedTheme: UserSelectedTheme): ComputedTheme {
+  if (userSelectedTheme === 'system') {
+    return getSystemTheme();
```

Comment:
This could be returned in one line.

Reply:
If we want to enforce as opposed to allow one line if statements, we should probably set up a lint rule for this. I can see this becoming harder to enforce as the team grows.